lib/kitchen/verifier/terraform.rb

```diff
@@ -50,6 +50,11 @@ def merge(options:)
       end
 
       def resolve_attributes(group:)
+        # discover output names
```

Comment:
This should be split in to two methods: loading default attributes and loading user attributes.